SYCL/Basic/alloc_pinned_host_memory.cpp

```diff
@@ -0,0 +1,46 @@
+// XFAIL: opencl || cuda
```

Comment:
Based on this check `// CHECK:ZE ---> zeMemAllocHost` this test will never pass on opencl or cuda. Is it possible to just not run this test on these backends?

Reply:
yes, it means that it will fail for opencl/cuda if we run. another option would be using REQUIRES: level_zero

Reply:
I think REQUIRES would be OK.
FYI @vladimirlaz

Reply:
I agree with Vlad

Reply:
done